docs/api/brl/brl_objectlist.md

```diff
@@ -4,6 +4,8 @@ title: BRL.ObjectList
 sidebar_label: Introduction
 ---
 
+## About
```

Comment:
You can't add anything to files within `docs/api` because they are auto-generated by processing the source files.
If you want to add something that will go here, you need to create/update the bbdoc file for this module.

Reply:
Feel free to copy the text and commit that, so I see what you mean? Which module has such a file so I see where to store and how to name?